src/eth_plugin_handler.c

```diff
@@ -64,7 +66,8 @@ void eth_plugin_prepare_query_contract_UI(ethQueryContractUI_t *queryContractUI,
     }
 
     queryContractUI->screenIndex = screenIndex;
-    strlcpy(queryContractUI->network_ticker, get_tx_network_ticker(), MAX_TICKER_LEN);
+    chain_id = get_tx_chain_id();
```

Comment:
Is it an error if get_tx_chain_id returns 0 (not found) ?

Reply:
It would be an error as that chain ID does not exist and can only be returned if it went through the `default` case of the `switch` statement in `get_tx_chain_id`.
But it has always been like this and this behavior hasn't changed, so it is out of the scope of this PR. :+1: